endtoend/evaluators/slashing.go

```diff
@@ -69,10 +69,10 @@ func validatorsLoseBalance(conns ...*grpc.ClientConn) error {
 	ctx := context.Background()
 	client := eth.NewBeaconChainClient(conn)
 
-	for i, indice := range slashedIndices {
+	for i, slashedIndex := range slashedIndices {
 		req := &eth.GetValidatorRequest{
 			QueryFilter: &eth.GetValidatorRequest_Index{
-				Index: indice,
+				Index: ethTypes.ValidatorIndex(slashedIndex),
```

Comment:
For consistency, let's rename e2e import alias to sth else, and have custom types imported as `types "github.com/prysmaticlabs/eth2-types"` -- as it has been done everywhere else.